### PR TITLE
Document inline validation

### PR DESCRIPTION
**Pull request by jo66**
## Subject

I am targeting this branch, because it's doc.

```markdown
### Added
- Added note to inline validation troubleshooting documentation.

### CODE REVIEW

**Comment by jo66** [comment]
It would have helped me, and I guess another newbies too.
But maybe troubleshooting should be moved or copied to CoreBundle documentation?

File: docs/reference/conditional_validation.rst [hunk 1]
```diff
@@ -42,3 +42,11 @@ entry, the whole $formOptions property or set validation_groups to an empty arra
     protected $formOptions = [
         'validation_groups' => []
     ];
+
+If there are some validation groups already configured that you want to keep, by
```
Comment (greg0ire) [inline docs/reference/conditional_validation.rst]:
```suggestion
If there are some validation groups already configured that you want to keep, for
```

**Comment by greg0ire** [comment]
>  But maybe troubleshooting should be moved or copied to CoreBundle documentation?

I really don't know… @core23 , maybe you do?

**Comment by core23** [comment]
> > But maybe troubleshooting should be moved or copied to CoreBundle documentation?
> 
> I really don't know… @core23 , maybe you do?

This problem has nothing to do with the core components, it's an admin problem

**Comment by greg0ire** [comment]
Thanks for the insight!

**Comment by greg0ire** [comment]
Thanks @jo66

**Comment by jo66** [comment]
@greg0ire You're welcome!

**Pull request merged**